models/shared/types/ownertype.go

```diff
@@ -23,7 +23,7 @@ func (o OwnerType) LocaleString(locale translation.Locale) string {
 	case OwnerTypeRepository:
 		return locale.TrString("concept_code_repository")
 	case OwnerTypeOrganization:
-		return locale.TrString("concept_user_organization")
+		return locale.TrString("organization")
```

Comment:
Why I used `concept_user_organization` here, the reason is the same as the previous discussion: I believe we should clearly define the concepts with context. I think "organization" could have different means in different contexts in different languages.

* One context, one concept, one translation.
* Different context, different concept, different translation.

Reply:
I increasingly think we need to restructure translations completely. Not group them by usage (new issue page, admin user table, installation page) but by context or meaning (`[permissions]`, `[operations]`, `[repo_units]` and so on)